index.html.tmpl

```diff
@@ -46,7 +46,11 @@
         url = decodeURIComponent(url[1]);
       } else {
         // servant-swagger-ui addition
-        url = "SWAGGER_URL_PLACEHOLDER";
+        var ssuDir = "SERVANT_SWAGGER_UI_DIR";
+        var ssuSchema = "SERVANT_SWAGGER_UI_SCHEMA";
+        url = window.location.pathname.replace(
+          "/" + ssuDir + "/",
```

Comment:
Yes, I think we can safely ignore backslashes.
However, you can't just replace `"/" + ssuDir + "/"`, since the same path piece can occur more than once in a location. You should do this instead:

```javascript
var re = new RegExp("\\/" + ssuDir + "(\\/(index\\.html)?)?$","g");
url = window.location.pathname.replace(re, "/" + ssuSchema);
```